yt_dlp/extractor/youtube.py

```diff
@@ -6927,6 +6941,37 @@ class YoutubeSearchURLIE(YoutubeTabBaseInfoExtractor):
             'id': 'youtube-dl test video',
             'title': 'youtube-dl test video',
         }
+    }, {
+        'url': 'https://www.youtube.com/results?baz=bar&search_query=IaSGqQa5O-M&filters=video&lclk=video',
```

Comment:
This isn't always reproducible, and yt-dlp ignores all these url params except the search query and `sp` (search params)

I would use `https://www.youtube.com/results?search_query=IaSGqQa5O-M&sp=EgIQAUICCAE%253D`

Reply:
One thing I'm wondering about is that the test case's timestamp would be even more inconsistent than what was done before putting everything into a single `approximate_metadata` arg (since I didn't add `approximate_date` to the test before), does that sound fine?

Reply:
since it is an approximate I would just test if its an `int` (i.e. some timestamp is extracted)